### PR TITLE
site: Contour v1.10.0 Blog Post

### PR DESCRIPTION
**Pull request by stevesloka**
Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

File: site/_posts/2020-11-06-contour_v1100.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+author_name: Steve Sloka
+author_avatar: /img/contributors/steve-sloka.png
+categories: [kubernetes]
+# Tag should match author to drive author pages
+tags: ['Contour Team']
+---
+
+Contour continues to add new features to help you better manage ingress operations in a cluster.
+Our latest feature release, Contour 1.10.0, now includes support for Envoy xDS v3 as support for the current v2 version will be deprecated in early 2021.
+Contour also adds support for multi-arch images allowing for deployment on platforms as well as extending support for custom JSON logging fields.
```
Comment (skriss) [inline site/_posts/2020-11-06-contour_v1100.md:14]:
```suggestion
Contour also adds support for multi-arch images allowing for deployment on multiple platforms as well as extending support for custom JSON logging fields. 
```

**Review by jonasrosland — APPROVED** [review]
This looks great, thank you @stevesloka !

File: site/_posts/2020-11-06-contour_v1100.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Multi-arch Images
+
+Similar to the new access logging features, users also have requested more architectures to run Contour.
+Envoy now supports ARM based architectures starting in Envoy v1.16.0 and Contour follows suit by offering multi-arch builds allowing Contour & Envoy to run on non amd64 based systems.
+
+![img][5]
+
+## Community Thanks!
+We’re immensely grateful for all the community contributions that help make Contour even better! For version 1.9, special thanks go out to the following contributors:
```
Comment (skriss) [inline site/_posts/2020-11-06-contour_v1100.md:53]:
```suggestion
We’re immensely grateful for all the community contributions that help make Contour even better! For version 1.10, special thanks go out to the following contributors:
```

**Review by skriss — APPROVED** [review]
LGTM

**Pull request merged**